third_party/2and3/requests/sessions.pyi

```diff
@@ -16,6 +16,7 @@ from . import adapters
 from . import status_codes
 
 BaseAdapter = adapters.BaseAdapter
+AuthBase = auth.AuthBase
```

Comment:
That's probably me guessing the wrong way to handle an error I was getting. :/ I initially tried using `type: Union[None, Tuple[Text, Text], auth.AuthBase, Callable[[Request], Request]] `, which gave me an error (`Invalid type "requests.sessions.Session.auth"`). My fix was presumably wrong -- do you know what the right one is? Thanks.